HabitTracker.harris-andy/Program.cs

```diff
@@ -0,0 +1,592 @@
+
+    using (var connection = new SqliteConnection(connectionString))
+    {
+        connection.Open();
+        for (int randomRecord = 0; randomRecord < 100; randomRecord++)
+        {
+            int randomIndex = random.Next(fakeActivities.Length);
+            string date = GetRandomDate();
+            string hobby = fakeActivities[randomIndex];
+            string units = fakeHobbies[hobby];
+            int quantity = random.Next(10);
+
+            using (var command = new SqliteCommand("INSERT INTO habits (Date, Hobby, Units, Quantity) VALUES (@date, @hobby, @units, @quantity)", connection))
+            {
+                command.Parameters.AddWithValue("@date", date);
+                command.Parameters.AddWithValue("@hobby", hobby);
+                command.Parameters.AddWithValue("@units", units);
+                command.Parameters.AddWithValue("@quantity", quantity);
+                command.ExecuteNonQuery();
+            }
+        }
+        connection.Close();
+    }
+}
+
+string GetRandomDate()
+{
+    Random random = new Random();
+    string day = Convert.ToString(random.Next(1, 31));
+    string month = Convert.ToString(random.Next(1, 13));
+    string year = Convert.ToString(random.Next(2023, 2025));
+    return $"{day.PadLeft(2, '0')}-{month.PadLeft(2, '0')}-{year}";
+}
+
+int validateNumberEntry(string message, bool isMainMenu = false)
+{
+    string? numberInput = null;
+    int finalInput = -1;
+    while (finalInput < 0)
+    {
+        Console.WriteLine(message);
+        numberInput = Console.ReadLine();
+        if (numberInput == "0")
+        {
+            if (isMainMenu)
+            {
+                return 0;
+            }
+            else
+            {
+                MainMenu();
+                return 1;
+            }
+        }
+        if (!int.TryParse(numberInput, out finalInput))
+        {
+            return -1;
+        }
+    }
+    return finalInput;
+}
+
+void DeleteTableContents()
+{
+    Console.Clear();
+    string? verify = null;
+    string[] prompts = {
+        "Are you SURE you want to delete all this data? y/n",
+        "Are you REALLY SUPER SURE? y/n",
+        "Last chance to turn back! Are you ABSOLUTELY TOTALLY POSITIVELY SURE? y/n"
+    };
+
+    foreach (string prompt in prompts)
+    {
+        Console.WriteLine(prompt);
+        verify = Console.ReadLine()?.ToLower();
+        if (verify != "y")
+        {
+            return;
+        }
+    }
+
+    using (var connection = new SqliteConnection(connectionString))
+    {
+        connection.Open();
+        using (var command = new SqliteCommand("DELETE FROM habits;", connection))
+        {
+            command.ExecuteNonQuery();
+        }
+        connection.Close();
+    }
+
+    Console.WriteLine("\n\n");
+    Console.WriteLine("Whelp. It's all gone. Now what...?");
+    Thread.Sleep(2000);
+    MainMenu();
+}
+class HobbyRecord

```

Comment:
I know the project allows for this. But I would really love to see this class in its own `HobbyRecord.cs` file outside of `Program.cs`